### ISSUE TITLE
Rename ILightCache Get and Set

### ISSUE THREAD

**Issue by bruno-brant** [ava-innersource/Liquid-Application-Framework-1.0-deprecated#177]
Method names conflict with reserved keywords an results in a warning ([CA1716](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1716?view=vs-2019)).

**Comment by guilhermeluizsp** [comment]
Do we really need synchronous versions of these methods? This smells like violation of ISP to me. If this interface were to be completely asynchronous, we could flawlessly implement its synchronous version.

**Comment by bruno-brant** [comment]
The methods have all been inherited from Redis implementation - that's the reason for the bad naming and the existence of both sync and async methods. 

Worse part is that different SDK's in other platforms might use different names (although Redis' command kinda suggest the name). 

In any case, I agree with you that the synchronous method is not required. We can remove it.

**Comment by PriscilaAvanade** [comment]
Shoud we really remove it or just turn it as obsolete?

**Comment by bruno-brant** [comment]
@PriscilaAvanade correct, we should always mark obsolete before deleting. Also, look at #107 and #108; We should be obsoleting the whole of `LightCache`, but not before enabling our clients to use `IDistributedCache`.
